### PR TITLE
Add unit test for NetworkConnect

### PR DESCRIPTION
**Pull request by vdemeester**
I was time for `NetworkConnect` to be tested 🐒.

🐸

Signed-off-by: Vincent Demeester <vincent@sbr.pm>

### CODE REVIEW

**Comment by calavera** [comment]
LGTM

File: client/network_connect_test.go [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+			}
+
+			return &http.Response{
+				StatusCode: http.StatusOK,
+				Body:       ioutil.NopCloser(bytes.NewReader([]byte(""))),
+			}, nil
+		}),
+	}
+
+	err := client.NetworkConnect("network_id", "container_id", &network.EndpointSettings{
```
Comment (duglin) [inline client/network_connect_test.go]: Should we test for cases where the EndpointConfig is nil to ensure that we don't see anything on the wire?

Comment (vdemeester) [inline client/network_connect_test.go]: Makes sense, I'll update.

Comment (calavera) [inline client/network_connect_test.go]: so...?

Comment (vdemeester) [inline client/network_connect_test.go]: Ah :scream_cat: I though I would remember to update after getting back from vacation.. will update right away !

**Comment by vdemeester** [comment]
Updated :wind_chime:

**Comment by duglin** [comment]
@vdemeester thanks - LGTM

**Pull request merged**